### DISCUSSION TITLE
[FEATURE] ReturnYouTubeDislike

### DISCUSSION THREAD

**Discussion by abdulrahman1s** [libre-tube/LibreTube#501] [General]
**ReturnYouTubeDislike API integration**
Bring back the dislike button.

**Additional context**
no

**Comment by lilcheti** [comment]
How should we do that

**Comment by alefvanoon** [comment]
@FireMasterK Do you have any plans to add it to piped-backend?

**Comment by FireMasterK** [comment]
No, the rate limit is too low and RYD's backend is closed source still.

**Comment by ghost** [comment]
You've probably already seen this, but there's a fork of NewPipe with SponsorBlock and ReturnYouTubeDislike - https://github.com/polymorphicshade/NewPipe. Might be helpful

**Comment by FireMasterK** [comment]
With https://github.com/TeamPiped/Piped-Backend/commit/374d067a9523e59de3c879d7090ff14a10976641, dislikes are no longer -1 in the API response!

**Comment by shuvashish76** [comment]
Also there is an app on F-Droid for this : https://f-droid.org/en/packages/com.jesperh.showyoutubedislikes/
Sourcecode : https://github.com/jesperbakhandskemager/view-youtube-dislike

**Comment by Silther** [comment]
Can this discussion be closed since the feature is implemented?
